src/devices/bus/nes/namcot.cpp

```diff
@@ -677,10 +675,23 @@ void nes_namcot163_device::write_h(offs_t offset, uint8_t data)
 		case 0x7800:
 			// the lower 4 bits work *BOTH* as WRAM write protect *AND* as sound address!
 			m_wram_protect = data & 0x0f;
-			LOG_MMC(("Namcot-163 sound address write, data: %02x\n", data));
+			m_namco163snd->addr_w(data);
 			break;
 		default:
 			n340_hiwrite(offset, data);
 			break;
 	}
 }
+
+//-------------------------------------------------
+//  device_add_mconfig - add device configuration
+//-------------------------------------------------
+
+void nes_namcot163_device::device_add_mconfig(machine_config &config)
+{
+	// additional sound hardware
+	SPEAKER(config, "addon").front_center();
```

Comment:
Does this cartridge actually have a separate sound output for the wavetable channels, or is it fed through the cartridge audio pins like the Konami VRC6/VRC7?  It should only have a `SPEAKER` here if it's an independent audio output.